packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.tsx

```diff
@@ -72,10 +76,34 @@ export class GuxTooltip {
     trackComponent(this.root, { variant: this.placement });
   }
 
+  componentDidLoad(): void {
+    if (!this.tooltipObserver && window.MutationObserver) {
+      this.tooltipObserver = new MutationObserver(
+        (): string => (this.role = this.baseTooltip?.role)
+      );
```

Comment:
I tried out the @OnMutation decorator but it didn't seem to pick up the role change on the base-tooltip.